content/tracing/product_specs/_index.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Product Specs
+kind: documentation
+---
+
+{{< whatsnext desc="Enclosed in this section are other articles that may be of interest:">}}
+    {{< nextlink href="/tracing/product_specs/metrics_namespace" >}} Metrics Namespace{{< /nextlink >}}
+    {{< nextlink href="/tracing/product_specs/trace_sampling_and_storage" >}} Trace Sampling and Storage{{< /nextlink >}}
+    {{< nextlink href="/tracing/product_specs/distributed_tracing" >}}Distributed Tracing Example{{< /nextlink >}}
```

Comment:
This doesn't feel like the right place.  This is more of a tutorial/getting started.